development-updates.md

```diff
@@ -393,6 +394,16 @@ Development updates from [`jsign`](https://github.com/jsign)
 
 [Link to Update #4](https://hackmd.io/@jsign/cohort-three-update-4)
 
+### Update #5
+
+[Link to Update #5](https://hackmd.io/@jsign/cohort-three-update-5)
```

Comment:
I'm not sure how this got lost, since it was merged https://github.com/eth-protocol-fellows/cohort-three/pull/110.
Maybe in some posterior PR somebody deleted it accidentally.